tests/implode_hamlet_256.bin

```diff
@@ -0,0 +1,2 @@
+
#67h���<D��������P!T�oՖK�Ѻt˺[V�Y�E��}��[� ��m˶,]� ��y5-[�i�7ȩhî-;nٰr���Mmڹ ˼7�7�V.H�h˂�;����6�[�e�����·��^�m�tM&Y�
c��t?c�����
��o8ٚ���>�@k6��|�t����~���A�l����_�
```

Comment:
Tag this (maybe in a .gitconfig file?) as a binary file. Otherwise the newline may get converted to Windows-style, breaking the test that uses this file.

Reply:
Wasn't so bad to use in code arrays for that after all? :)

Reply:
Update: chose my .gitattributes file over yours because it covers entire directories and is therefore more future-proof.